unblob/handlers/__init__.py

```diff
@@ -27,6 +27,7 @@
     yaffs,
 )
 from .filesystem.android import sparse
+from .filesystem.netgear import chk
```

Comment:
I would move it to `archive` rather than `filesystem`. CHK has no concept of filesystem.

Reply:
I changed the location of to `netgear` under the folder `chk` to mark the difference with chk and trx